src/time-picker/tests/unit/TimePicker.spec.tsx

```diff
@@ -65,32 +65,33 @@ const baseTemplate = (date?: Date) =>
 		);
 	});
 
-const buttonTemplate = assertionTemplate(() => {
-	return (
-		<div classes={css.input}>
-			<TextInput
-				disabled={undefined}
-				key="input"
-				focus={() => false}
-				theme={{}}
-				onBlur={noop}
-				onFocus={noop}
-				onValue={noop}
-				initialValue={''}
-				onValidate={noop}
-				required={undefined}
-				valid={undefined}
-				helperText=""
-				onKeyDown={noop}
-				type="text"
-				variant={undefined}
-				classes={undefined}
-			>
-				{{ label: undefined, trailing: undefined }}
-			</TextInput>
-		</div>
-	);
-});
+const buttonTemplate = (required?: boolean) =>
```

Comment:
test looks good, but we've tried really hard to remove any need for these `getRender` / `getTemplate` functions via the addition of the assertion template so I'd rather we didn't add one here when you could simply to

```
const requiredTemplate = baseTemplate
			.setProperty('@input', 'required', true)
```